src/plugins/prefs/README.md

```diff
@@ -0,0 +1,107 @@
+- infos = Information about the prefs plugin is in keys below
+- infos/author = Name <name@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = storage
+- infos/recommends =
+- infos/placements = getstorage setstorage
+- infos/status = maintained reviewed conformant compatible coverage specific tested nodep libc preview experimental difficult unfinished nodoc concept 
+- infos/metadata =
+- infos/description =
+
+## Basics ##
+
+### Preference Types ###
+
+- Default preferences: below `mountpoint/preferences/pref/`.
+- User preferences: below `mountpoint/preferences/user/`.
+- Lock preferences: below `mountpoint/preferences/lock/`.
+
+User preferences will override default preferences. Lock preferences can't be overwritten by user or default preferences. 
+Only user preferences will be saved to `prefs.js`.
```

Comment:
What do you mean here? Does this storage write to multiple files or is it relevant how the file is named?